control_plane/src/bin/neon_local.rs

```diff
@@ -1046,10 +1047,20 @@ fn get_pageserver(env: &local_env::LocalEnv, args: &ArgMatches) -> Result<PageSe
     ))
 }
 
+fn get_retry_timeout(args: &ArgMatches) -> &Duration {
```

Comment:
nit: let's call the method like the flag & variable, i.e., `get_start_timeout`

Reply:
ok